test/inductor/test_torchinductor.py

```diff
@@ -6544,8 +6544,12 @@ def test_full_truncation(self):
         def fn(a):
             return a + torch.full_like(a, 7.777)
 
+        device_interface = get_interface_for_device(self.device)
         for dtype in all_types():
-            self.common(fn, (make_tensor(8, dtype=dtype, device=self.device),))
+            if device_interface.is_dtype_supported(dtype):
+                self.common(fn, (make_tensor(8, dtype=dtype, device=self.device),))
+            else:
+                self.assertRaises(TypeError)
```

Comment:
Question is: do we want to test that types that are not supported are not really supported? If not, we can just skip running any tests for those dtypes. If we do, I propose something along those lines
```python
   ctx = context.nullcontext() if device_interface.is_dtype_supported(dtype) else self.assertRaises(TypeError)
   with ctx:
      self.common(...)
```
Also, please note that if one iterates over all dtypes, it's probably better to invoke common with `check_lowp=False` otherwise it will always run 2nd round of tests, where input tensor is cast to float16
```suggestion
```